flink-table/flink-table-planner-blink/pom.xml

```diff
@@ -149,20 +135,28 @@ under the License.
 			<version>${calcite.version}</version>
 			<exclusions>
 				<!--
-				"mvn dependency:tree" as of Calcite 1.21:
+				"mvn dependency:tree" as of Calcite 1.22.0:
 
-				[INFO] +- org.apache.calcite:calcite-core:jar:1.21.0:compile
-				[INFO] |  +- commons-codec:commons-codec:jar:1.10:compile
-				[INFO] |  +- org.apache.calcite:calcite-linq4j:jar:1.21.0:compile
-				[INFO] |  +- org.apache.commons:commons-lang3:jar:3.3.2:compile
-				[INFO] |  +- com.fasterxml.jackson.core:jackson-core:jar:2.10.1:compile
+				[INFO] +- org.apache.calcite:calcite-core:jar:1.22.0:compile
+				[INFO] |  +- org.apache.calcite:calcite-linq4j:jar:1.22.0:compile
 				[INFO] |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.10.1:compile
-				[INFO] |  +- com.fasterxml.jackson.core:jackson-databind:jar:2.10.1:compile
-				[INFO] |  +- com.google.guava:guava:jar:19.0:compile
-				[INFO] |  \- com.jayway.jsonpath:json-path:jar:2.4.0:compile
+				[INFO] |  +- org.apiguardian:apiguardian-api:jar:1.1.0:compile
+				[INFO] |  +- com.fasterxml.jackson.core:jackson-core:jar:2.10.1:runtime
+				[INFO] |  +- com.fasterxml.jackson.core:jackson-databind:jar:2.10.1:runtime
+				[INFO] |  +- com.jayway.jsonpath:json-path:jar:2.4.0:runtime
+				[INFO] |  |  \- net.minidev:json-smart:jar:2.3:runtime
```

Comment:
Did you check the resulting JAR file? Are all these dependencies relocated? Can we exclude them?

Reply:
We can not exclude them. All the jars listed here are needed by Calcite code.

The `org.apache.commons:commons-lang3` and `commons-io:commons-io` are not in the shade file set, should i add them in ?

Reply:
I added these 2 to the relocation part.

Reply:
I'm not sure whether `commons-lang3` should be relocated, because it was also in the dependency tree and the version is the same to the one in `flink-core`.

Reply:
commons-lang3 is a conflict prone jar, we should shade it. So user that depends on flink would not have conflict.

Reply:
I have removed the relocation of `commons-lang3` because all the other modules in Flink does not do that, although it is indeed a conflict prone jar, we can do that uniformly if we think it is necessary in the future.